contributors/guide/scalability-good-practices.md

```diff
@@ -102,68 +96,62 @@ func (c *ControllerX) Run() {
  
-If you're changing some big and important dependency, there's no way of knowing what performance impact it'll have. The only thing we can do is to actually run test and check.
+If you are changing a large and important dependency, there is no way of knowing what performance impact it will have. We can only run test and check.
 Where to look to get data?
-If you want to check the impact of your changes there are a number of places you can look at:
-Density/Load tests output quite a lot of data either to test logs, or files inside 'ReportDir' - both of them include API call latencies and Density also includes pod e2e startup latency information,
-for resource usage you can either use monitoring tools (heapster + graphana, but at the time of writing it stops working around 100 Nodes), or just plain old top on master (which scales as much as you want:),
-more data is available on `/metrics` endpoint on all our components (e.g. one for the API server contains API call latencies),
-to profile a component is to create an ssh tunnel to the machine running it, and run `go tool pprof localhost:<your_tunnel_port>` locally
+If you want to check the impact of your changes there are a number of places where you can look:
+Density/Load tests output quite a lot of data either to test logs, or files inside 'ReportDir' - both of them include API call latencies, and Density also includes pod e2e startup latency information.
+for resource usage you can either use monitoring tools (heapster + graphana, but at the time we wrote this it stops working at around 100 Nodes), or just plain 'top' on master (which scales as much as you want),
+more data is available on the `/metrics` endpoint of all our components (e.g. the one for the API server contains API call latencies),
+to profile a component create an ssh tunnel to the machine running it, and run `go tool pprof localhost:<your_tunnel_port>` locally
 
 ## Summary
-Summing it up, when writing code you should:
+To summarize, when writing code you should:
 - understand how go manages memory and use it wisely,
-- not LIST from API server,
-- run performance tests in case of huge system-wide changes (e.g. updating big dependencies),
+- not LIST from the API server,
+- run performance tests when making large systemwide changes (e.g. updating big dependencies),
 
-When designing new features/thinking about refactoring you should:
-- when adding new API calls estimate number of additional QPS you'll be sending to the API server,
-- when adding complex logic make sure that you don't add it on a critical path of any basic workflow
+When designing new features or thinking about refactoring you should:
+- when adding new API calls estimate the number of additional QPS you will be sending to the API server,
```

Comment:
no comma

Reply:
I removed the command and reorganized the sentence structure of the bullet points to make for a smoother read.